### PR TITLE
ci: publishing dart-3.1.2 failed

### PR DESCRIPTION
**Pull request by RodrigoSMarques**
### New Pull Request Checklist
- [X] I am not disclosing a [vulnerability](https://github.com/parse-community/Parse-SDK-Flutter/security/policy).
- [X] I am creating this PR in reference to an [issue](https://github.com/parse-community/Parse-SDK-Flutter/issues?q=is%3Aissue).

### Issue Description
Remove dependency_overrides in flutter/pubspec.yaml

Related issue: #771 

### Approach
N/A

### TODOs before merging
N/A

- [ ] Add tests
- [ ] Add changes to documentation (guides, repository pages, in-code descriptions)
- [ ] A changelog entry

### CODE REVIEW

File: packages/flutter/pubspec.yaml [hunk 1]
```diff
@@ -28,9 +28,6 @@ dependencies:
   sembast: ^3.2.0
   sembast_web: ^2.0.1+1
   path: ^1.8.1 # required for transitive use only
-
-dependency_overrides:
-  path: ^1.8.2 # required for transitive use only
   platform: ^3.1.0
```
Comment (mtrezza) [inline packages/flutter/pubspec.yaml:31]: should platform also be removed?

Comment (RodrigoSMarques) [reply packages/flutter/pubspec.yaml:31]: não. foi removido de dependency_overrides e deixado apenas em dependency

Comment (RodrigoSMarques) [reply packages/flutter/pubspec.yaml:31]: not.  was removed from dependency_overrides and left only in dependency

Comment (mtrezza) [reply packages/flutter/pubspec.yaml:31]: got it

**Review by mtrezza — APPROVED** [review]

**Pull request merged**